src/Runner.Common/JobServer.cs

```diff
@@ -169,6 +172,11 @@ private void InitializeWebsocketClient(TimeSpan delay)
                     Trace.Info($"Creating websocket client ..." + feedStreamUrl);
                     this._websocketClient = new ClientWebSocket();
                     this._websocketClient.Options.SetRequestHeader("Authorization", $"Bearer {accessToken}");
+                    var userAgentValues = new List<ProductInfoHeaderValue>();
+                    userAgentValues.AddRange(UserAgentUtility.GetDefaultRestUserAgent());
```

Comment:
`UserAgentUtility.GetDefaultRestUserAgent()` was from VSSF, it's less interesting to the Runner itself, `HostContext.UserAgents` should have all the useful part we want.

Reply:
kay am going to leave it as it for consistency then